stock_picking_auto_create_lot_biochousa/oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+stock-logistics-workflow
```

Comment:
Wrong location for `oca_dependencies.txt`: it should be placed in repository root folder

Reply:
I still see this file in the wrong path